src/screens/teams/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { LeaderBoard } from './components/Leaderboard';
+
+import { DefaultLayout } from '~/shared/layouts/default-layout';
+
+export default function TeamsScreen() {
+  return (
+    <DefaultLayout>
+      <LeaderBoard.Container title="League Of Legends LEC">
+        <LeaderBoard.Teams
```

Comment:
I think it'd make more sense to rename `LeaderBoard` to `Team` and `LeaderBoard.Teams` to `Team.Leaderboard`